### PR TITLE
Return to 2 account pages

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

This PR updates the account page view back to 2 pages for now, overview page and the tokens page.
This is so we can create a new release soon enough, and build gradually the new account pages one by one.

Also fixes the issues mentioned #850 except for the point 2

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @nenadV91 , these changes look nice to me!

Some issues:
1. Affiliate banners look like tiles now on the Account page, so the page layout is changed now. See images: 
affiliate message 2
affiliate message

Compare with Prod: 
cards layout

This causes an issue with cards layout: Cow balance and Governance cards are huge now: https://watch.screencastify.com/v/ISMInRF3Sp6ssbERpH7S

Could you please fix it? 

Thanks

**Comment by elena-zh** [comment]
Hey @nenadV91 , great changes!

Some minor issues: 
1. Maybe we can move the scrollbar closer to the boarder of the modal, so it will be displayed in the same position as in the list below, WDYT?
move like here
2. 'Affiliate program' section name is missing on the General tab
section name
image
3. Do not have a strong opinion here, but frim my perspective it would be nice to make the page name in black/light blue like we do for all the rest pages in the app (plus like it is currently on Prod) instead of the orange color
image
See the About page as an example 
image
image
4. Mobile + not connected + Mainnet: content of the affiliate section exceeds the container
image
(issue with the Connect button is reported in #669)
5. We disabled Sell button to a not connected user and when a token is not approved, however, all the rest buttons are always enabled. Maybe we should keep all buttons enabled all the time, WDYT?

Thanks!

**Comment by nenadV91** [comment]
@elena-zh 
1. Fixed
2. Fixed
3. Very good suggestion, fixed
4. on hold for now
5. good suggestion, fixed

**Comment by elena-zh** [comment]
Hey @nenadV91 , great!
All the issues I mentioned above are fixed (even 4th case).
The last nitpick: it would be nice if notification text does not exceed the notification area. Reproducible when a user is not connected to a wallet in a mobile view in iOS
image
Could you please take a look at it?

File: src/custom/components/Tokens/TokensTableRow.tsx [hunk 1]
```diff
@@ -100,13 +102,24 @@ const DataRow = ({
   const isPendingOnchainApprove = approvalState === ApprovalState.PENDING
   const isPendingApprove = !isApproved && (approving || isPendingOnchainApprove)
 
-  const noBalance = !balance || balance?.equalTo(0)
-  const noAllowance = !currentAllowance || currentAllowance.equalTo(0)
+  const hasZeroBalance = !balance || balance?.equalTo(0)
+  const hasNoAllowance = !currentAllowance || currentAllowance.equalTo(0)
+
+  // This is so we only create fiat value request if there is a blance
```
Comment (alfetopito) [inline src/custom/components/Tokens/TokensTableRow.tsx:108]: Typo

```suggestion
  // This is so we only create fiat value request if there is a balance
```

**Comment by elena-zh** [comment]
I have just noticed that Manage tokens modal has become broken in the PR 
image

File: src/custom/pages/Account/index.tsx [hunk 2]
```diff
@@ -29,6 +34,23 @@ function _getPropsFromRoute(route: string) {
   }
 }
 
+// Note: As we build these single pages, we will remove this component in the future
+const Overview = () => {
+  const { chainId } = useWeb3React()
+
+  return (
+    <Container>
+      {chainId && chainId === ChainId.MAINNET && <AffiliateStatusCheck />}
```
Comment (alfetopito) [inline src/custom/pages/Account/index.tsx:43]: Nitpick: If chainId is undefined the check will fail against mainnet anyway

```suggestion
      {chainId === ChainId.MAINNET && <AffiliateStatusCheck />}
```

**Review by alfetopito — APPROVED** [review]
Very nice 👏

**Comment by nenadV91** [comment]
@elena-zh Both issues you mentioned should be fixed now, nice catch on the modal issue :pray:

**Comment by elena-zh** [comment]
Hey @nenadV91  , changes LGTM now.. 
Besides how the General page looks in iPad mini. See images:
ipad
ipda
Could we please do anything with this?

**Review by elena-zh — APPROVED** [review]
Hey @nenadV91 , the last fix helped to fix the issue with iPad layout for the landscape mode, but not for Portrait. 
But as we have found out, the issue is reproducible in Prod, and it not related to the current PR changes. So I created #979 issue for this to address it later. 
Also, I have created another low-prio iOS-related issue #980 . 

I' approving the current PR. Great job!

**Comment by alfetopito** [comment]
Merging onto https://github.com/cowprotocol/cowswap/pull/932 to review it

**Pull request merged**